app/wp-content/themes/kstg/library/scss/layout/_vehicle.scss

```diff
@@ -1,16 +1,24 @@
 .vehicle {
-  padding: 3.3%;
+  padding: 0 5% 80px;
   background-color: $light-grey;
-  h2 {
-    font-size: 1.6em;
-    margin: 25px 0;
+  clear: both;
+  .container {
```

Comment:
@sarahlshowers: I wouldn't use a mixin, just move anything that's generic to `core/_base.scs` perhaps? So in that file it would probably be something like:

```css
max-width: 1220px;
width: 95%;
margin: 0 auto;
box-sizing: border-box;
&.content {
  padding:  80px 8% 45px;
  background-color: #FFF;
  border: 1px solid $medium-grey;
  }
}
```

And then just override padding and not specific to vehicle in this file.